pykafka/simpleconsumer.py

```diff
@@ -194,7 +194,7 @@ def start(self):
 
         # Figure out which offset wer're starting on
         if self._reset_offset_on_start:
-            self._reset_offsets()
+            self.reset_offsets()
             # make sure the reset is saved in kafka before it can rebalance
```

Comment:
Ah yes, that's indeed what I meant - but I think that last line needs to be wrapped in `if self._consumer_group is not None`, because otherwise `commit_offsets` might raise an exception?